src/main/java/org/pac4j/vertx/auth/Pac4jUser.java

```diff
@@ -76,15 +76,15 @@ public void writeToBuffer(Buffer buff) {
 
     @Override
     public int readFromBuffer(int pos, Buffer buffer) {
-        pos = super.readFromBuffer(pos, buffer);
-        final int jsonByteCount = buffer.getInt(pos);
-        pos += 4;
-        final byte[] jsonBytes = buffer.getBytes(pos, pos + jsonByteCount);
-        pos += jsonByteCount;
+        int posLocal = super.readFromBuffer(pos, buffer);
```

Comment:
When using another variable for the position, shouldn't the `final` keyword be used for the `int pos` input? I mean:

```
public int readFromBuffer(final int pos, final Buffer buffer) {
  int posLocal = super.readFromBuffer(pos, buffer);
```